akd_client/Cargo.toml

```diff
@@ -21,10 +22,11 @@ winter-utils = { version = "0.2", optional = true }
 sha2 = { version = "0.10.1", optional = true, default-features = false }
 sha3 = { version = "0.10.0", optional = true, default-features = false }
 blake3 = { version = "1.3.1", optional = true, default-features = false }
-serde_json = "1"
 
 wasm-bindgen = { version = "0.2.79", optional = true, features = ["serde-serialize"] }
-serde = { version = "1.0", features = ["derive"]}
+serde = { version = "1.0", optional = true, features = ["derive"] }
+serde_json = { version = "1", optional = true}
+protobuf = { version = "=2.8.1", optional = true }
```

Comment:
Since there is forward-/backward-compatibility in protobuf, why not use 2.8.1? We get the improvements in the library.